test/redshift_test.py

```diff
@@ -29,7 +29,7 @@
 from luigi.s3 import S3Client
 
 
-if sys.version_info[:2] == (3, 4):
+if (3, 4, 0) <= sys.version_info[:3] < (3, 4, 3):
```

Comment:
Backward incompatibility in python 3.4 that was fixed in 3.4.3 http://bugs.python.org/issue20386.